### ISSUE TITLE
Smart Heater "You should enable 'tempReporting.....'

### ISSUE THREAD

**Issue by tjenvy** [bwp91/homebridge-govee#388]
### What issue do you have? Please be as thorough and explicit as possible.
Homebridge is spitting out a notice every 30 seconds stating:
"[12/14/2022, 10:22:08 AM] [Govee] [Smart Heater] you should enable `tempReporting` in the config for this device."
This just started recently.  
I followed an existing issue https://github.com/bwp91/homebridge-govee/issues/354 instructions but never got the option.  I also manually added the config item per the comments and it still keeps saying it.  

Twi questions:
1.. What is this?
2.. In homebridge what do I need to do to fix this. 

### Details of your setup.
* Do you use Homebridge (with config-ui?) or HOOBS? 
Homebridge

* Which version of Homebridge/HOOBS do you have?
1.6.0

* Which version of this plugin (homebridge-govee) do you have? Has the issue started since upgrading from a previous version?
7.5.1

* Which Govee devices do you have that are causing issues? Please include product models if applicable.
Smart Heater Model Name	H7130

### Please paste any relevant logs below.
[12/14/2022, 10:25:49 AM] [Govee] [Smart Heater] you should enable `tempReporting` in the config for this device.

```

```

**Comment by bwp91** [comment]
Hi @tjenvy 

can you show your current govee config here so I can add the option for you? Remember to remove your email/password/api-key

**Comment by bwp91** [comment]
> 1.. What is this?

It means that the device can report the ambient temperature to homebridge which enables showing the device as a HomeKit heater accessory rather than a fan. It's much better 😃

**Comment by tjenvy** [comment]
`"name": "Govee",
            "username": "",
            "password": "",
            "debug": false,
            "platform": "Govee"`

**Comment by tjenvy** [comment]
> 

Oh man that is way better!

**Comment by bwp91** [comment]
```json
{
   "name": "Govee",
   "username": "",
   "password": "",
   "debug": false,
   "heaterDevices": [
     {
       "label": "My Heater (this can be anything)",
       "deviceId": "11:22:33:44:55:66:77:88",
       "tempReporting": true
      }
   ],
   "platform": "Govee"
}
```

Try changing your config to this (don't forget to re-enter your username, password) and also change the deviceId of the heater to yours (and update the label if wanted)

If you're not familiar with editing the config like this (using JSON) then please check the final `}`, put a comma after it (like `},` if there is already a comma there in your own config

**Comment by tjenvy** [comment]
Same error message.  

[12/14/2022, 2:34:05 PM] [Govee] [Smart Heater] you should enable `tempReporting` in the config for this device.


        {
            "name": "Govee",
            "username": "",
            "password": "",
            "debug": false,
            "heaterDevices": [
                {
                    "label": "Nicks Heater",
                    "deviceId": "",
                    "tempReporting": true
                }
            ],
            "platform": "Govee"
        }

**Comment by bwp91** [comment]
where are you getting your device id from?

**Comment by tjenvy** [comment]
It’s amazing what 1 digit does LOL I fixed it.  It’s working now and showing correctly!

After adding that bracketed section I noticed that the main giver plugging page in home bridge had the heater there with the temperature reporting option.  It was strange because that wasn’t there before.  Previously it was jsut enter the info > add then it disappeared from that screen and the second tab opened.  All good now though! Thanks!
